src/ML-SNAP/compute_snap.cpp

```diff
@@ -231,8 +252,9 @@ void ComputeSnap::init()
   if (force->pair == nullptr)
     error->all(FLERR,"Compute snap requires a pair style be defined");
 
-  if (cutmax > force->pair->cutforce)
+  if (cutmax > force->pair->cutforce){
```

Comment:
See my comment above about programming style. That includes not adding curly braces for a single statement. If you do add them, then there should be a blank before the curly brace. The following page summarizes the most important conventions:
https://docs.lammps.org/Modify_style.html#programming-style-requirements-varied